src/chunk.rs

```diff
@@ -0,0 +1,105 @@
+use std::iter::*;
+use std::fmt::Debug;
+
+/// Chunk of code text
+#[derive(Debug)]
+pub enum Chunk {
+    Text(String),
+    EndLine,
+}
```

Comment:
They may not be good at handling the long lines currently, but I believe it more productive to improve the existing formatters.